src/lib/frontend/components/CategoryProducts.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import type {OfferSummaryPresentation} from "$lib/frontend/presentations/OfferSummaryPresentation";
```

Comment:
Remind me the difference between the CategoryProducts and the Collection

Reply:
Collection contains unrelated products without any actions. CategoryProducts contains products from the same category with pagination behaviour (loadMore)

Reply:
Where do we have Collections in the app right now?

Reply:
Home page